ui/src/app/views/workspace-edit/component.ts

```diff
@@ -330,4 +330,11 @@ export class WorkspaceEditComponent implements OnInit {
   set workspacePopulationDetails(val) {
     this.workspace.researchPurpose.underservedPopulationDetails = val;
   }
+
+  get allowSave() {
+    if (this.savingWorkspace) {
+      return true;
```

Comment:
This should return `false` since you don't want to allow saving in this case; template should negate the response.